components/timezone/src/ixdtf.rs

```diff
@@ -910,7 +923,12 @@ impl FromStr for DateTime<Iso> {
 }
 
 impl<A: AsCalendar> DateTime<A> {
-    /// Creates a [`DateTime`] in any calendar from an IXDTF syntax string with compiled data.
+    /// Creates a [`DateTime`] in any calendar from an IXDTF syntax string.
+    ///
+    /// Returns an error if the string has a calendar annotation that does not
+    /// match the calendar argument.
+    ///
+    /// ✨ *Enabled with the `ixdtf` Cargo feature.*
```

Comment:
the type itself has this annotation, we don't put it on every method.

Reply:
This is on DateTime which is not itself gated.